code/datums/components/embedded.dm

```diff
@@ -303,7 +303,9 @@
 		return
 	var/damage = weapon.w_class * remove_pain_mult
 	limb.receive_damage(brute=(1-pain_stam_pct) * damage * 1.5, sharpness=SHARP_EDGED) // Performs exit wounds and flings the user to the caster if nearby
-	limb.force_wound_upwards(/datum/wound/pierce/bleed/moderate)
+	var/datum/wound/wound_typepath = get_corresponding_wound_type(list(WOUND_PIERCE), limb, WOUND_SEVERITY_MODERATE)
+	if (wound_typepath)
+		limb.force_wound_upwards(wound_typepath)
```

Comment:
Already discussed in discord but I'd like to see the boilerplate reduced on this to be on par with how it was prior